plugins/woo-ai/includes/attribute-suggestion/class-attribute-suggestion-prompt-generator.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * Woo AI Attribute Suggestion Prompt Generator Class
+ *
+ * @package Woo_AI
+ */
+
+namespace Automattic\WooCommerce\AI\AttributeSuggestion;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Attribute Suggestion Prompt Generator Class
+ */
+class Attribute_Suggestion_Prompt_Generator {
+	public const SYSTEM_PROMPT = <<<SYSTEM_PROMPT
+You are a SEO and marketing expert specializing in e-commerce stores built using WooCommerce.
+
+You are given the product's name, description, tags, categories, and other attributes. You will also be given a requested attribute.
+Your task is to provide three optimized alternatives to the requested attribute to enhance the online store's SEO performance and sales.
+Suppose the requested attribute is the name. In that case, you will provide the best possible option for the product's name based on the other attributes, such as the given product name, description, tags, and categories.
+
+Only return suggestions for the requested attribute in the "content" part of your JSON response. The request will be in the following format, the value of each attribute will be inside double quotation marks:
+
+Requested Attribute:
+Name:
+Description:
+Tags (comma separated):
+Categories (comma separated, child categories separated with >):
+
+Structure your response as JSON (RFC 8259), similar to this example:
```

Comment:
I don't want to over-engineer but it feels like the JSON response aspect of the system response is going to become a very common request. Should we break this out into something more reusable?